.gitignore

```diff
@@ -7,6 +7,9 @@
 /spec/reports/
 /tmp/
 
+# environment variables
+.env*
```

Comment:
@mscottford I think it is a good practice to include a `.env.sample` when you add `dotenv` as a dependency. Later on we can add a `bin/setup` that helps new contributors get started with the project. 

It would be great if after merging this PR we could end up with instructions for new contributors that just said:

```
bin/setup
rspec
```

To run the build for the first time.